pkgs/applications/networking/wgnord/default.nix

```diff
@@ -55,12 +55,13 @@ resholve.mkDerivation rec {
     ];
   };
 
-  meta = with lib; {
+  meta = {
     description = "A NordVPN Wireguard (NordLynx) client in POSIX shell";
     homepage = "https://github.com/phirecc/wgnord";
     changelog = "https://github.com/phirecc/wgnord/releases/tag/v${version}";
-    maintainers = with lib.maintainers; [ urandom ];
```

Comment:
Why this?

Reply:
Because I removed the `with lib:`

tracking issue: https://github.com/NixOS/nixpkgs/issues/292468

Reply:
Is `with lib.maintainers;` also bad?

Reply:
Depends, but having a `with lib;` scope and then `with maintains;` is a nested `with` scope.

But for my shake writing `with lib.maintainers` would also be fine.

Reply:
Then don't change it here. I want the diff to be as small as possible and relevant to the problem being solved.

Reply:
Minimized the diff now. Thank you for your time (: